sdk/scheduler/src/test/resources/test-update-pod-a.yml

```diff
@@ -0,0 +1,36 @@
+name: "test"
```

Comment:
Is there anything like Python's [factory_boy](https://factoryboy.readthedocs.io/en/latest/) for creating fixtures in code and automating the generation of valid values for fields you don't care about in a test? I hate fixtures with a passion, and if we can nip this proliferation of slightly-differing YAMLs now, all the better.